src/main/java/com/mattermost/integration/figma/webhook/WebhookController.java

```diff
@@ -10,9 +13,24 @@
 @RequestMapping("/webhook")
 public class WebhookController {
 
+    private final FileNotificationService fileNotificationService;
+
+    public WebhookController(FileNotificationService fileNotificationService) {
+        this.fileNotificationService = fileNotificationService;
+    }
 
     @PostMapping("/comment")
-    public void comment(@RequestBody String request) {
-        System.out.println("Got it "+ request);
+    public void comment(@RequestBody FileCommentWebhookResponse response) {
+        System.out.println(response);
+        if (!response.getEventType().equals("PING")) {
```

Comment:
Are we sure that EventType will be always a part of request?
If not please consider to rewrite like: "PING".equals(response.getEventType()) in this case if  response.getEventType() = null we will get just false result instead of NullPointerException